### ISSUE TITLE
Attribute order matters ?

### ISSUE THREAD

**Issue by Awea** [slime-lang/slime#93]
Hi there, I've recently imported some slim template from a Ruby project, at first everything was fine. Then something happens ! 

I write a test in `test/rendering/attributes_test.exs` to make this more understable:

``` elixir
test "attributes order doesn't matter" do
  assert render("a#bar.foo") == ~s(<a class="foo" id="bar"></a>) # => true
  assert render("a.foo#bar") == ~s(<a class="foo" id="bar"></a>) # => false
end
```

**Comment by lpil** [comment]
Are you suggesting we sort the order of the attributes? I don't see the advantage of this.

**Comment by Awea** [comment]
@lpil no. The second test fail with this ouput:

```
lhs:  "<a class=\"foo\"></a>"
rhs:  "<a class=\"foo\" id=\"bar\"></a>"
```

So in the current implementation of Slime if the id is after a class declaration it's ignored ?

**Comment by lpil** [comment]
Ah, I see. Yes, this is a bug.

**Comment by Awea** [comment]
An update on `@tag_regex` should fix it right ? I'm always confused by the regexs :D

**Comment by doomspork** [comment]
Closed as part of #94.  Released as `0.12.2`
